e2e/protractor.conf.js

```diff
@@ -4,14 +4,15 @@
 const { SpecReporter } = require('jasmine-spec-reporter');
 
 exports.config = {
-  allScriptsTimeout: 11000,
+  allScriptsTimeout: 77000,
```

Comment:
too big better to have timeout for a test

Reply:
changed